From: Andrew Morton <akpm@linux-foundation.org>
Subject: memcg-manually-uninline-__memcg_memory_event-fix
Date: Mon Oct 27 04:58:56 PM PDT 2025

use EXPORT_SYMBOL_GPL for __memcg_memory_event, per Michal and Christoph

Cc: Johannes Weiner <hannes@cmpxchg.org>
Cc: Michal Hocko <mhocko@suse.com>
Cc: Muchun Song <muchun.song@linux.dev>
Cc: Roman Gushchin <roman.gushchin@linux.dev>
Cc: SeongJae Park <sj@kernel.org>
Cc: Shakeel Butt <shakeel.butt@linux.dev>
Cc: Christoph Hellwig <hch@infradead.org>
Signed-off-by: Andrew Morton <akpm@linux-foundation.org>
---

 mm/memcontrol.c |    2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

--- a/mm/memcontrol.c~memcg-manually-uninline-__memcg_memory_event-fix
+++ a/mm/memcontrol.c
@@ -1655,7 +1655,7 @@ void __memcg_memory_event(struct mem_cgr
 	} while ((memcg = parent_mem_cgroup(memcg)) &&
 		 !mem_cgroup_is_root(memcg));
 }
-EXPORT_SYMBOL(__memcg_memory_event);
+EXPORT_SYMBOL_GPL(__memcg_memory_event);
 
 static bool mem_cgroup_out_of_memory(struct mem_cgroup *memcg, gfp_t gfp_mask,
 				     int order)
_
